src/Features/CSharp/Portable/MakeRefStruct/CSharpMakeRefStructCodeFixProvider.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.CSharp.MakeRefStruct
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp), Shared]
+    internal class CSharpMakeRefStructCodeFixProvider : CodeFixProvider
```

Comment:
nit: in general the `CSharp` or `VisualBasic` prefix is for when there are multiple versions of this provided for both languages.  That said, we probably don't consistently follow that rule for other C#-only features, so i'm ok with leaving this as is.

Reply:
OK, thx, will consider this in the next PR's 😄